src/code/reducers/index.js

```diff
@@ -31,6 +31,63 @@ function initialState() {
           });
 }
 
+function fertilize(state) {
```

Comment:
This was a general question I had - what is the purpose of this `fertilize` function? I see that we call it in the code, but I'm not sure what it does differently than `BioLogica.breed`. You can see that I used the same logic below to breed random clutches, but I wonder if I can replace that logic with calls to `BioLogica`